ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/MainnetDifficultyCalculators.java

```diff
@@ -67,15 +69,20 @@ private MainnetDifficultyCalculators() {}
         return periodCount > 1 ? adjustForPeriod(periodCount, difficulty) : difficulty;
       };
 
+  @VisibleForTesting
   public static DifficultyCalculator<Void> BYZANTIUM =
       (time, parent, protocolContext) ->
-          calculateByzantiumDifficulty(time, parent, BYZANTIUM_FAKE_BLOCK_OFFSET);
+          calculateThawedDifficulty(time, parent, BYZANTIUM_FAKE_BLOCK_OFFSET);
 
-  public static DifficultyCalculator<Void> CONSTANTINOPLE =
+  static DifficultyCalculator<Void> CONSTANTINOPLE =
       (time, parent, protocolContext) ->
-          calculateByzantiumDifficulty(time, parent, CONSTANTINOPLE_FAKE_BLOCK_OFFSET);
+          calculateThawedDifficulty(time, parent, CONSTANTINOPLE_FAKE_BLOCK_OFFSET);
 
-  private static BigInteger calculateByzantiumDifficulty(
+  static DifficultyCalculator<Void> MUIR_GLACIER =
+      (time, parent, protocolContext) ->
+          calculateThawedDifficulty(time, parent, MUIR_GLACIER_FAKE_BLOCK_OFFSET);
+
+  private static BigInteger calculateThawedDifficulty(
```

Comment:
Could this be `calculateMuirGlacierDifficulty`? As it is it seems like it reduces code discoverability if you're searching for instances of the hard fork, like you could have when it was `Byzantium`?

Reply:
It's used in 3 different places, byzantium, constantinople, and now MuirGlacier.  Before it was `calculageByzantiumDifficulty`.